test/model.test.js

```diff
@@ -725,6 +725,78 @@ describe('Model', function () {
 
     });   // End of '$pull modifier'
 
+    describe('$max modifier', function () {
+      it('Will set the field to the updated value if value is greater than current one, without modifying the original object', function () {
+        var obj = { some:'thing', number: 10}
+            , updateQuery = {$max:{number:12}}
```

Comment:
nedb uses a more spaced style for objects, here for example it would be `{ $max: { number:12 } }`